builtin/game/chatcommands.lua

```diff
@@ -147,11 +147,12 @@ core.register_chatcommand("help", {
 core.register_chatcommand("privs", {
 	params = "<name>",
 	description = "print out privileges of player",
-	func = function(name, param)
-		param = (param ~= "" and param or name)
-		return true, "Privileges of " .. param .. ": "
+	func = function(caller, param)
+		param = param:gsub("%s", "")
```

Comment:
:+1: to @HybridDog's comment.

With this code, `/privs sin gle pla yer` has the same effect as `/privs singleplayer`. While it's unlikely this will cause any permanent damage, it's a bit inconsistent.